mappings/com/mojang/blaze3d/systems/RenderSystem.mapping

```diff
@@ -253,30 +253,36 @@ CLASS com/mojang/blaze3d/systems/RenderSystem
 		ARG 2 width
 		ARG 3 height
 	CLASS class_5590 IndexBuffer
-		FIELD field_27332 sizeMultiplier I
-		FIELD field_27333 increment I
+		COMMENT An index buffer that automatically grows as needed.
+		FIELD field_27332 vertexCountForShape I
+		FIELD field_27333 indexCountForShape I
 		FIELD field_27334 indexMapper Lcom/mojang/blaze3d/systems/RenderSystem$class_5590$class_5591;
 		FIELD field_27335 id I
 		FIELD field_27336 indexType Lnet/minecraft/class_293$class_5595;
 		FIELD field_27337 size I
 		METHOD <init> (IILcom/mojang/blaze3d/systems/RenderSystem$class_5590$class_5591;)V
-			ARG 1 sizeMultiplier
-			ARG 2 increment
+			ARG 1 vertexCountForShape
+				COMMENT the number of vertices needed to form a single shape
+			ARG 2 indexCountForShape
+				COMMENT the number of indices needed to form a single shape
 			ARG 3 indexMapper
 		METHOD method_31920 grow (I)V
-			ARG 1 newSize
+			ARG 1 requiredSize
 		METHOD method_31922 getIndexConsumer (Ljava/nio/ByteBuffer;)Lit/unimi/dsi/fastutil/ints/IntConsumer;
-			ARG 1 indicesBuffer
+			ARG 1 indexBuffer
 		METHOD method_31923 (Ljava/nio/ByteBuffer;I)V
 			ARG 1 index
 		METHOD method_31924 getIndexType ()Lnet/minecraft/class_293$class_5595;
 		METHOD method_31925 (Ljava/nio/ByteBuffer;I)V
 			ARG 1 index
-		METHOD method_43409 isSizeLessThanOrEqual (I)Z
-			ARG 1 size
+		METHOD method_43409 isLargeEnough (I)Z
+			ARG 1 requiredSize
 		METHOD method_43410 bindAndGrow (I)V
-			ARG 1 newSize
+			COMMENT Binds this buffer as a current index buffer. If necessary, it grows this
+			COMMENT buffer in size and uploads indices to the corresponding buffer in GPU.
+			ARG 1 requiredSize
 		CLASS class_5591 IndexMapper
```

Comment:
```suggestion
		CLASS class_5591 Triangulator
```
The process of converting a shape into triangles is called [triangulation](https://en.wikipedia.org/wiki/Polygon_triangulation) (confusingly, but it is the correct terminology)